common-compose/build.gradle

```diff
@@ -53,7 +53,7 @@ dependencies {
     androidTestImplementation 'androidx.test.ext:junit:1.1.5'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.5.1'
 
-    def composeBom = platform('androidx.compose:compose-bom:2024.01.00')
+    def composeBom = platform('dev.chrisbanes.compose:compose-bom:2024.03.00-alpha01')
```

Comment:
if you merge https://github.com/zumba/consumer-app-android/pull/369 please remember to match versions